src/components/Results/ResultsTable.tsx

```diff
@@ -55,33 +36,31 @@ function ResultsTable() {
           </TableRow>
         </TableHead>
         <TableBody>
-          {Array(10)
-            .fill(0)
-            .map((_, index) => {
-              const {
-                platformName,
-                testName,
-                baseValue,
-                newValue,
-                delta,
-                confidenceText,
-                baseRuns,
-                newRuns,
-              } = results[0];
-              return (
-                <TableRow key={index}>
-                  <TableCell>{platformName}</TableCell>
-                  <TableCell>{testName}</TableCell>
-                  <TableCell>{baseValue}</TableCell>
-                  <TableCell>{newValue}</TableCell>
-                  <TableCell>{delta}%</TableCell>
-                  <TableCell>{confidenceText}</TableCell>
-                  <TableCell>
-                    {baseRuns}/{newRuns}
-                  </TableCell>
-                </TableRow>
-              );
-            })}
+          {results.map((result, index) => {
+            const {
+              platformName,
+              testName,
+              baseValue,
+              newValue,
+              delta,
+              confidenceText,
+              baseRuns,
+              newRuns,
+            } = result;
+            return (
+              <TableRow key={index}>
+                <TableCell>{platformName}</TableCell>
+                <TableCell>{testName}</TableCell>
+                <TableCell>{baseValue}</TableCell>
+                <TableCell>{newValue}</TableCell>
+                <TableCell>{delta}%</TableCell>
+                <TableCell>{confidenceText}</TableCell>
```

Comment:
[suggestion] do you want to use a map to iterativelly generate the cells?

Reply:
I considered this, but was unsure what we will be doing in the future, so mapping may not work. example, for delta, when we have `is_improvement` and `is_regression`, we'll likely use conditional rendering to display either a check icon or x icon.